pkg/kubectl/resource/builder_test.go

```diff
@@ -843,7 +846,7 @@ func TestIgnoreStreamErrors(t *testing.T) {
 	singular := false
 
 	err := b.Do().IntoSingular(&singular).Visit(test.Handle)
-	if err != nil || singular || len(test.Infos) != 2 {
+	if err == nil || singular || len(test.Infos) != 2 {
```

Comment:
@feihujiang, sorry I don't have time to review it today. I'll try to review it tomorrow. @deads2k feel free to grab this one if you have time.